### PR TITLE
DOC-122: LocationBookmarks JSDoc

### PR DESCRIPTION
**Pull request by ctrlaltdavid**
Namepaces:
- LocationBookmarks

Case: https://highfidelity.atlassian.net/browse/DOC-122

Note: No executable code changes.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20435/HighFidelity-Beta-PR15951-c2b67f2.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20435/HighFidelity-Beta-Interface-PR15951-c2b67f2.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20435/HighFidelity-Beta-PR15951-c2b67f2.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20435/HighFidelity-Beta-Interface-PR15951-c2b67f2.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20435/HighFidelity-Beta-PR15951-c2b67f2.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20435/HighFidelity-Quest-Beta-PR15951-c2b67f2.apk)

File: interface/src/LocationBookmarks.h [hunk 1]
```diff
@@ -35,28 +38,35 @@ class LocationBookmarks : public Bookmarks, public  Dependency  {
     static const QString HOME_BOOKMARK;
 
     /**jsdoc
+     * Gets the metaverse address associated with a bookmark.
      * @function LocationBookmarks.getAddress
-     * @param {string} bookmarkName Name of the bookmark to get the address for.
-     * @returns {string} The url for the specified bookmark. If the bookmark does not exist, the empty string will be returned.
+     * @param {string} bookmarkName - Name of the bookmark to get the metaverse address for. {Case sensitive.)
```
Comment (jennaingersoll) [inline interface/src/LocationBookmarks.h]:
```suggestion
     * @param {string} bookmarkName - Name of the bookmark to get the metaverse address for. (Case sensitive)
```

File: interface/src/LocationBookmarks.h [hunk 2]
```diff
@@ -35,28 +38,35 @@ class LocationBookmarks : public Bookmarks, public  Dependency  {
...
+     * @returns {string} The metaverse address for the bookmark. If the bookmark does not exist, <code>""</code> is returned.
+     * @example <caption>Report the "Home" bookmark's metaverse address.</caption>
+     * print("Home bookmark's address: " + LocationBookmarks.getAddress("Home"));
      */
     Q_INVOKABLE QString getAddress(const QString& bookmarkName);
 
 public slots:
 
     /**jsdoc
+     * Prompts the user to bookmark their current location: the user can specify the name of the bookmark in the dialog that is
```
Comment (jennaingersoll) [inline interface/src/LocationBookmarks.h]:
```suggestion
     * Prompts the user to bookmark their current location. The user can specify the name of the bookmark in the dialog that is 
```

File: interface/src/Bookmarks.h [hunk 3]
```diff
@@ -52,6 +52,7 @@ class Bookmarks: public QObject {
 
 protected slots:
     /**jsdoc
+     * Prompts the user to delete a bookmark: the user can select the bookmark to delete in the dialog that is opened.
```
Comment (jennaingersoll) [inline interface/src/Bookmarks.h]:
```suggestion
     * Prompts the user to delete a bookmark. The user can select the bookmark to delete in the dialog that is opened.
```

File: interface/src/LocationBookmarks.h (hunk 2 above)
Comment (ctrlaltdavid) [reply interface/src/LocationBookmarks.h]: Done.

File: interface/src/Bookmarks.h (hunk 3 above)
Comment (ctrlaltdavid) [reply interface/src/Bookmarks.h]: Done.

File: interface/src/LocationBookmarks.h (hunk 1 above)
Comment (ctrlaltdavid) [reply interface/src/LocationBookmarks.h]: Changed to "... for (case sensitive)."

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20456/HighFidelity-Beta-PR15951-ddffb8a.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20456/HighFidelity-Beta-Interface-PR15951-ddffb8a.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20456/HighFidelity-Beta-PR15951-ddffb8a.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20456/HighFidelity-Quest-Beta-PR15951-ddffb8a.apk)

**Comment by ctrlaltdavid** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20461/HighFidelity-Beta-PR15951-ddffb8a.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20461/HighFidelity-Beta-Interface-PR15951-ddffb8a.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20461/HighFidelity-Beta-PR15951-ddffb8a.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20461/HighFidelity-Quest-Beta-PR15951-ddffb8a.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20461/HighFidelity-Beta-PR15951-ddffb8a.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20461/HighFidelity-Beta-Interface-PR15951-ddffb8a.exe

File: interface/src/LocationBookmarks.h [hunk 4]
```diff
@@ -40,7 +40,7 @@ class LocationBookmarks : public Bookmarks, public  Dependency  {
     /**jsdoc
      * Gets the metaverse address associated with a bookmark.
      * @function LocationBookmarks.getAddress
-     * @param {string} bookmarkName - Name of the bookmark to get the metaverse address for. {Case sensitive.)
+     * @param {string} bookmarkName - Name of the bookmark to get the metaverse address for {case sensitive).
```
Comment (jennaingersoll) [inline interface/src/LocationBookmarks.h]: should be a ( before "case"

Comment (ctrlaltdavid) [reply interface/src/LocationBookmarks.h]: Fixed

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20465/HighFidelity-Beta-PR15951-1e8a205.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20465/HighFidelity-Beta-Interface-PR15951-1e8a205.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20465/HighFidelity-Beta-PR15951-1e8a205.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20465/HighFidelity-Quest-Beta-PR15951-1e8a205.apk)

**Comment by jennaingersoll** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20466/HighFidelity-Beta-PR15951-1e8a205.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20466/HighFidelity-Quest-Beta-PR15951-1e8a205.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20466/HighFidelity-Beta-PR15951-1e8a205.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20466/HighFidelity-Beta-Interface-PR15951-1e8a205.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20466/HighFidelity-Beta-PR15951-1e8a205.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20466/HighFidelity-Beta-Interface-PR15951-1e8a205.exe

**Pull request merged**